dojo/fixtures/test_type.json

```diff
@@ -719,6 +719,13 @@
         },
         "model": "dojo.test_type",
         "pk": 194
+    },
+    {
+        "fields": {
+            "name": "Wfuzz JSON Report"
+        },
+        "model": "dojo.test_type",
+        "pk": 195
```

Comment:
you don't need this modification anymore

Reply:
Need to modify the new parser doc so to reflect this